app/models/candidate_interface/work_experience_form.rb

```diff
@@ -22,7 +22,7 @@ class WorkExperienceForm
     validates :role, :organisation,
               length: { maximum: 60 }
 
-    validates :details,
+    validates :details, :working_pattern,
```

Comment:
Could we add a better validation message for when it appears at the top of the page?

<img width="679" alt="Screenshot 2020-02-05 at 10 09 50" src="https://user-images.githubusercontent.com/642279/73832262-bd840580-47ff-11ea-92cc-b532fcd4ca37.png">

Reply:
Good catch

Reply:
![image](https://user-images.githubusercontent.com/22743709/73833120-55ceba00-4801-11ea-9e8d-fc40a54a09dd.png)

@fofr @duncanjbrown Are you happy with this? following the same pattern that skills and experience has.

Reply:
I've add a small content tweak